features/contract-state/Totals.tsx

```diff
@@ -103,20 +124,24 @@ const Totals = () => {
               of <White>collateral</White> supplied
             </Label>
             <LinksContainer>
-              <SmallLink
-                href={getEtherscanUrl(collAddress)}
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                Etherscan
-              </SmallLink>
-              <SmallLink
-                href={getExchangeLinkCollateral}
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                {exchangeName}
-              </SmallLink>
+              {collAddress && (
```

Comment:
You should pass in `collAddress`  into `renderComponent` so that this file is more functional. And you should check `collAddress !== null` like you do with `getExchangeLinkColalteral` because it is a string type.

Reply:
I can do that. if we make that change then we should do the same for `collAddress` as well as `tokenAddress` to be consistent. I did not do that with either to be consistent with the previous implementation but am happy to add it.

Reply:
Yeah it's easy enough to pass in and make the change to be consistent